terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/sqs.ts

```diff
@@ -1,21 +1,23 @@
 import { ActionRequestMessage } from './scale-up';
-import { SQS } from 'aws-sdk';
+import { ChangeMessageVisibilityBatchCommand, SendMessageBatchCommandInput, SQS } from '@aws-sdk/client-sqs';
 import { SQSRecord } from 'aws-lambda';
 
 import { expBackOff } from './utils';
 import { Metrics } from './metrics';
 
 function getQueueUrl(evt: SQSRecord, sqs: SQS) {
   const splitARN = evt.eventSourceARN.split(':');
+  // arn:aws:sqs:region:account-id:queue-name
+  const region = splitARN[3];
   const accountId = splitARN[4];
   const queueName = splitARN[5];
-  return sqs.endpoint.href + accountId + '/' + queueName;
+  return `https://sqs.${region}.amazonaws.com/${accountId}/${queueName}`;
```

Comment:
endpoint.href no longer seems to be a thing, so I construct it manually, hopefully it's correct.  There's also a getQueueUrl function but it returns string | undefined